x-pack/plugins/cross_cluster_replication/public/app/_app.scss

```diff
@@ -0,0 +1,19 @@
+/**
+ * 1. Encapsulate this fix to avoid modifying global styling.
+ */
+.ccrPageContent {
+  max-width: 1000px !important;
+  width: 100% !important;
+
+  .euiText .euiButton--warning {
+    color: $euiColorWarning; // [1]
```

Comment:
Why is this color being overridden here?

There are two problems with this:

1. EUI classes shouldn't be directly overridden (even with scoping). If this is a bug in EUI, please just make an issue in EUI and we'll fix it there.
2. We take care to ensure that our text colors meet accessibility contrast levels and the current `$euiColorWarning` as text on any background does not. In EUI, we use color functions to alter the color until it does meet these contrast levels so by overriding these you are no longer compliant.

Reply:
Thanks for pointing this out, it's definitely something we should fix in EUI. https://github.com/elastic/eui/issues/1372